pkgs/development/libraries/libcef/default.nix

```diff
@@ -0,0 +1,84 @@
+{ stdenv, mkChromiumDerivation, fetchgit, gtkglext
+, fontconfig, freetype, cairo, pango, libX11, libXi, libXext
+, libXfixes, libXrender, libXcomposite, dbus, atk, libXrandr
+, libXmu, libXt, libSM, libICE }:
+
+with stdenv.lib;
+
+mkChromiumDerivation (base: rec {
+  name = "libcef";
+  packageName = "cef";
+  # Run "git rev-list --count HEAD" to get this number
+  cefVersion = "1414";
+  # from CHROMIUM_BUILD_COMPATIBILITY.txt
+  upstreamVersion = "51.0.2704.47";
```

Comment:
Okay, so it probably might make sense to add it to the updater. However I'm not sure on how to namespace it properly, because if the chromium updater also updates `libcef` it might get confusing, so it's probably better to move the updater out of the `chromium` directory, as it's not only used for Chromium, but also for Chrome and now `libcef`, so we should pick an appropriate name for it.

But we can still decide on that at some later point.